systran_storages/storages/generic.py

```diff
@@ -176,12 +176,12 @@ def stream_corpus_manager(self, remote_id, remote_format, buffer_size=1024):
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def push_file(self, local_path, remote_path):
+    def push_file(self, local_path, remote_path, lp):
```

Comment:
Is there any warning in S3, Swift storage about this change of function argument?